codebase/src/java/disco/org/openlvc/disco/pdu/radio/SignalPdu.java

```diff
@@ -114,13 +114,13 @@ public void to( DisOutputStream dos ) throws IOException
 	@Override
 	public final int getContentLength()
 	{
-		return 19 + data.length;
+		return 20 + data.length;
 		
 		// int size = entityID.getByteLength();              // 6
 		// size += DisSizes.UI16_SIZE;	// Radio ID          // 2
 
 		// size += encodingScheme.getByteLength();           // 2
-		// size += TdlType.getByteLength();                  // 1
+		// size += TdlType.getByteLength();                  // 2
```

Comment:
TdlType was being treated as an 8-bit value but is actually a 16-bit value:

Per IEEE 1278.1 s 7.7.3.e
> TDL Type. This field shall specify the TDL Type and shall be represented as a 16-bit enumeration
> (see [UID 178]). Detailed requirements for setting this field are specified in 5.8.4.3.2